DVR/HTTPURLResponse.swift

```diff
@@ -1,18 +1,18 @@
 import Foundation
 
-// There isn't a mutable NSHTTPURLResponse, so we have to make our own.
-class URLHTTPResponse: NSHTTPURLResponse {
+// There isn't a mutable HTTPURLResponse, so we have to make our own.
+class HTTPURLResponse: Foundation.HTTPURLResponse {
```

Comment:
You can directly reference via the module, that is, either `DVR.HTTPURLResponse` or `Foundation.HTTPURLResponse`.

I think having it that way is most explicit and readable.